### ISSUE TITLE
Feature Request: Keyword Alert

### ISSUE THREAD

**Issue by itstake** [DrKLO/Telegram#905]
please add Keyword alert feature

how it works:

1. a set keyword 'asdf'
2. a turn off b's chat room alert
3.if b chat 'eqwe', telegram will not alert b's message, but b chat 'asdf', telegram will alert b's message

sorry for bad english :(

**Comment by vmkazakoff** [comment]
Agree with this function! Will be very good idea, when I can mute chat, but receive notification if one of keyword is used. May be it can be merged with hashtag.

next idea: make some sort of "chat" based on hashtag (I know, I can click hashtag and find all messages, but first I need find a hashtag).

**Comment by kauron** [comment]
There is already an alert feature, if you mention someone `@name` then he/she will get alerted even if the chat was muted.

**Comment by vmkazakoff** [comment]
@kauron, yes, I know about this feature, but what we with @itstake mean - be notified when special keyword (not a user name) used.

For example - I have a chat with 100+ colleagues. If I want read messages about marketing updates I have 3 way:
1) read all chat about all updates for Company (100+ users - almost impossible even if this is "no flood" chat), but sometime I need all this info - usually I read it once a week all 1000 messages at once, but this in not enough if I want stay tuned about something.
2) use hashtag, and several times in a day scroll last messages and look for this hashtag, then press it and read all I need
3) ask all colleagues mention me and all other colleagues if they send message about marketing, but this will make messages too long with all mentions. Example @user1, @user2, ... @user38 - *your messages goes here*. All mentions will take more space than a text.

What I see as feature to do - make a option to add my "favorite hashtag" and receive notifications if they are used. Or, if hashtag notification is too hart to implement, make hashtags star'able - if I click on hashtag, I can press some "star" button and see it in chat list.

Hope now I explain correctly my idea. Thanks!
